sw/device/silicon_creator/rom/bootstrap.c

```diff
@@ -261,15 +261,15 @@ static rom_error_t bootstrap_handle_erase(bootstrap_state_t *state) {
 static rom_error_t bootstrap_handle_erase_verify(bootstrap_state_t *state) {
   HARDENED_CHECK_EQ(*state, kBootstrapStateEraseVerify);
 
-  rom_error_t err_0 = flash_ctrl_data_erase_verify(0, kFlashCtrlEraseTypeBank);
-  rom_error_t err_1 = flash_ctrl_data_erase_verify(
-      FLASH_CTRL_PARAM_BYTES_PER_BANK, kFlashCtrlEraseTypeBank);
-  HARDENED_RETURN_IF_ERROR(err_0);
-  HARDENED_RETURN_IF_ERROR(err_1);
+  // rom_error_t err_0 = flash_ctrl_data_erase_verify(0,
```

Comment:
@alphan can you think of a good way where we can let DV bypass the erase verification stage?
These checks below take up basically most of the test time.

I wouldn't be opposed to using an OTP bit.. :)

Reply:
this change is currently causing unittest to fail.

Reply:
What if we add a `#define OT_TEST_ROM` or something similar so that this is removed at compile time?

Reply:
Unfortunately not at the moment. We have one more use-case and this will require some thought.

@moidx, I think this should be DV/simulation specific, not test_rom since we would want to perform this check on the FPGA to keep the bootstrap as close to the actual thing as possible.

Reply:
yes DV / simulation specific is fine. The problem is that there is no DV specific test rom, but even if there were, everyone right now invokes the actuaul rom's "bootstrap", so we all end up in the same place.

That's why i think having an OTP bit would be better.  A zero value of the otp should always do the verified read, whereas a hardened non-zero value would skip it.  This would make it harder for an attacker to arbitrarily glitch into that state.

Reply:
OK but since this is DV/sim specific, I don't think an OTP entry is the right approach. I'll keep this in mind when working on this.